src/generator/AutoRest.Go/TransformerGo.cs

```diff
@@ -136,7 +136,7 @@ private void TransformModelTypes(CodeModelGo cmg)
                     name = $"{name}Type";
                 }
 
-                if (CodeNamerGo.Instance.UserDefinedNames.Contains(name))
+                if (CodeNamerGo.UserDefinedNames.Contains(name))
```

Comment:
In this specific case you're right it doesn't really matter.  However, there was a push to eliminate static fields and methods as they cannot be virtual (although it's likely these will never need to be virtual).  It feels like unnecessary churn IMO unless there's a real motivation for it that I'm missing.